controller/src/command.rs

```diff
@@ -396,9 +405,11 @@ where
 	let mut receive_compressed = true;
 	if method == "emoji" {
 		(slate, receive_compressed) = EmojiSlate().decode(&args.input.as_str())?;
+	} else if method == "qr" {
+		slate = QrToSlate((&args.input).into()).get_tx()?;
 	} else {
 		slate = PathToSlate((&args.input).into()).get_tx()?;
-	}
+	};
```

Comment:
We should transform this into a match statement. Maybe we also could transform the method into a enum?

Reply:
I don't know if it needs to be an Enum, the QR method was based on the File method codes, but the writing functions are different.

The reading function can even be generalized.